backbone.ipynb

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
<p>Overall the layer name composition and channel dimension parsing in line 81-84 feels super brittle, but I guess they always work for <code>ResNet</code>-based  models?</p><p>We might want to add tests to verify the correctness for this function since we have so many hardcoded assumptions here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yycho0108/ai604-video-object-pose/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>